hack/util.sh

```diff
@@ -464,6 +464,28 @@ function util::delete_all_clusters() {
   echo "Deleted all clusters and the log file is in ${log_file}"
 }
 
+# util::delete_necessary_resources deletes clusters(karmada-host, member1, member2 and member3) and related resources directly
+# util::delete_necessary_resources actually do three things: delete cluster、remove kubeconfig、record delete log
+# Parameters:
+#  - $1: KUBECONFIG files of clusters, separated by ",", such as "~/.kube/karmada.config,~/.kube/members.config"
+#  - $2: clusters, separated by ",", such as "karmada-host,member1"
+#  - $3: log file path, such as "/tmp/karmada/"
+function util::delete_necessary_resources() {
+  local config_files=${1}
+  local clusters=${2}
+  local log_path=${3}
+
+  local log_file="${log_path}"/delete-necessary-resources.log
+  rm -rf ${log_file}
+  mkdir -p ${log_path}
+  local config_file_arr=$(echo ${config_files}| tr ',' ' ')
+  local cluster_arr=$(echo ${clusters}| tr ',' ' ')
+
+  rm -f ${config_file_arr}
```

Comment:
why above `rm -rf ${log_file}` has a `-r` option, while this line `rm -f ${config_file_arr}` not.

Is there any purpose?

Reply:
There is no special intention, just to be consistent with the previous function.
https://github.com/karmada-io/karmada/blob/e7192a9ecc7bd01f1ef103411bdb78a5cd8fcddc/hack/util.sh#L458-L469

Reply:
I think it is not a good habit, if it which you removed is a dirtory, append `-r`; if it is just a file, no need.